docs/revisions.md

```diff
@@ -2,6 +2,18 @@
 
 (ZIP-Archives including APK, readme + profiles)
 
+### New last version
+
+Android
+
+- Add parameter dialog for profile
+
+Library
```

Comment:
Unrelated to this PR. I think we should stick to a common format like [keepachangelog](https://keepachangelog.com/) which suggests a *unreleased* section instead of "new last version" (which should be "new *since* last version"?)